sdks/python/test-suites/direct/py3/build.gradle

```diff
@@ -29,19 +29,9 @@ task postCommitIT(dependsOn: 'installGcpTest') {
   doLast {
     def batchTests = [
         "apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it",
-        "apache_beam.examples.cookbook.bigquery_tornadoes_it_test:BigqueryTornadoesIT.test_bigquery_tornadoes_it",
```

Comment:
@Juta, why are we removing some tests here?

Reply:
I noticed that all IT tests are only included in the DataflowRunner and not in the DirectRunner in the python 2 tests. I made the changes to exactly match the tests from python 2: https://github.com/apache/beam/blob/master/sdks/python/build.gradle#L101

Reply:
Ok, thank you. We can keep the subsets of Direct runner tests matching for the purposes of this PR between Py2 and Py3 suites.

Reply:
On the other hand, these subsets will become out of sync unless we find a way to put these tests in some" DirectRunner postcommit IT" suite, and run all tests from the suite. 
@markflyhigh , do direct runner tests run in parallel to DF runner tests? Are there any concerns with running all ITs with direct runner in the postcommit?

Reply:
Some tests do not run on the DirectRunner in py2 and py3 because of an issue that @pabloem recently pointed out in dev@beam.apache.org:
"The DirectRunner does not seem to support RuntimeValueProvider. Is there a suggestion for DirectRunner pipelines to access arguments passed in as pipeline options(but not necessarily passed explicitly by users) at pipeline execution time? ... I'm working on a transform that writes to big query, and table destinations can come in the form "dataset.table", or "project:dataset.table". Because these are parsed at runtime (destinations are dynamic), the ptransform checks if a project was provided in pipelineoptions via value providers."

Reply:
Sorry about the trouble, but this will be fixed by https://github.com/apache/beam/pull/8093

Reply:
And furthermore, I did not realize these tests were broken because they werent running on DirectRunner, so I see the value of having at least one or two BQIO tests on Directrunner.

Reply:
Good point. I think unless there are specific reasons not to run some ITs on direct runner, we should run them all in postcommit suite.

Reply:
I think we can merge this, and enable all ITs for direct runner both on Py2 and Py3 in a follow-up change.

Reply:
They are running in parallel with postCommitIT and VR tests. see https://scans.gradle.com/s/slw53cu2vk7n6 for example. I think changes here looks good to me.